### PR TITLE
add verification view

### PR DESCRIPTION
**Pull request by pxpm**
This PR should be merged when https://github.com/Laravel-Backpack/CRUD/pull/5301 gets merged.

image

### CODE REVIEW

**Comment by tabacitu** [comment]
Eeh.... too wide for my taste 👀 

Why isn't this card narrower, like the one in the Tabler PR?

**Comment by pxpm** [comment]
image

**Comment by tabacitu** [comment]
Checked in browser too. Looks good now, thanks Pedro. Good to go imo!

File: resources/views/auth/verify-email.blade.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
+@extends(backpack_view('layouts.plain'))
+
+@section('content')
+
+<div class="row justify-content-center">
+    <div class="col-12 col-md-8 col-lg-4">
```
Comment (tabacitu) [inline resources/views/auth/verify-email.blade.php:6]:
```suggestion
    <div class="col-12 col-md-8 col-lg-6">
```

File: resources/views/auth/verify-email.blade.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        <h3 class="text-center mb-4">{{ trans('backpack::base.verify_email.email_verification') }}</h3>
+        <div class="card">
+            <div class="card-body">
+            {{ trans('backpack::base.verify_email.email_verification_required') }}
+
+            @if (session('status') == 'verification-link-sent')
+                <div class="mb-4 mt-2">
+                    <div class="alert alert-success" role="alert">
```
Comment (tabacitu) [inline resources/views/auth/verify-email.blade.php:14]:
```suggestion
                    <div class="alert alert-success my-0" role="alert">
```

File: resources/views/auth/verify-email.blade.php [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+<div class="row justify-content-center">
+    <div class="col-12 col-md-8 col-lg-4">
+        <h3 class="text-center mb-4">{{ trans('backpack::base.verify_email.email_verification') }}</h3>
+        <div class="card">
+            <div class="card-body">
+            {{ trans('backpack::base.verify_email.email_verification_required') }}
+
+            @if (session('status') == 'verification-link-sent')
+                <div class="mb-4 mt-2">
```
Comment (tabacitu) [inline resources/views/auth/verify-email.blade.php:13]:
```suggestion
                <div class="mt-2">
```

**Review by tabacitu — CHANGES_REQUESTED** [review]

**Pull request merged**